source/common/stats/tag_extractor_impl.cc

```diff
@@ -294,5 +294,14 @@ bool TagExtractorTokensImpl::searchTags(const std::vector<absl::string_view>& in
   return pattern_index == tokens_.size() && input_index == input_tokens.size();
 }
 
+TagExtractorFixedImpl::TagExtractorFixedImpl(absl::string_view name, absl::string_view value)
+    : TagExtractorImplBase(name, value), value_(std::string(value)) {}
+
+bool TagExtractorFixedImpl::extractTag(TagExtractionContext&, std::vector<Tag>& tags,
+                                       IntervalSet<size_t>&) const {
+  addTag(tags) = value_;
```

Comment:
the name `addTag()` makes me think of a function with a side effect. It looks weird that it returns a mutable reference that is then assigned to.

Reply:
Agreed -- this was pre-existing but I'm renaming to addTagReturningValueRef.